packages/block-editor/src/components/block-navigation/style.scss

```diff
@@ -74,7 +81,7 @@ $tree-item-height: 36px;
 
 	.block-editor-block-mover-button {
 		width: $button-size;
-		height: $button-size;
+		height: 16px;
```

Comment:
Movers PR should supersede these customizations

Reply:
Do you mean this one? https://github.com/WordPress/gutenberg/pull/22673